contribute.md

```diff
@@ -1,9 +1,17 @@
 # Contribute
 
-This page is a stub.
+## How to Contribute
 
-For now, please check out the following links:
+Get in touch with us
+[joining our Discord](https://discord.gg/PQFdubGt6d),
+[opening a thread in our forum](https://forum.vac.dev/),
+or [opening issues / PRs on GitHub](https://github.com/vacp2p).
```

Comment:
Probably makes sense to also link back to Waku contribute page https://waku.org/contribute

Might want to link to "good first issues too" - @jm-clius I know you marked a few this way?

Reply:
Indeed! https://github.com/status-im/nwaku/labels/good%20first%20issue